GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -603,14 +582,11 @@ GPUd() bool GPUTRDTracker_t<TRDTRK, PROP>::ConvertTrkltToSpacePoint(const GPUTRD
   xTrkltDet[1] = trklt.GetY();
   xTrkltDet[2] = pp->GetRowPos(trkltZbin) - pp->GetRowSize(trkltZbin) / 2.f - pp->GetRowPos(pp->GetNrows() / 2);
   matrix->LocalToMaster(xTrkltDet, xTrkltSec);
-  sp.mR = xTrkltSec[0];
-  sp.mX[0] = xTrkltSec[1];
-  sp.mX[1] = xTrkltSec[2];
-  sp.mDy = trklt.GetdY();
-
-  int modId = geo.GetSector(det) * GPUTRDGeometry::kNstack + geo.GetStack(det); // global TRD stack number
-  unsigned short volId = geo.GetGeomManagerVolUID(det, modId);
-  sp.mVolumeId = volId;
+  sp.setX(xTrkltSec[0]);
```

Comment:
In principle, there is no strict need that the SpacePoint is a class.
In O2 it is also allowed that we have structs, particularly when all members are accessible directly anyway, and when there is no complicated functionality inside.
In that case, you would not need the setters / getters. But it is up to you.

Reply:
In O2 `GPUTRDSpacePoint` is the same as `o2::trd::CalibratedTracklet`, which is a class. For AliRoot I needed the same interface, therefore I created a class. Of course we could also change `CalibratedTracklet` to be a simple struct. But since the class does not create any overhead I think it is fine to leave it the way it is